packages/core/declare.d.ts

```diff
@@ -1,2 +1,3 @@
 declare module 'http-range-fetcher'
 declare module 'load-script2'
+declare module 'jexl'
```

Comment:
`@types/jexl` is installed, so I don't think we should need to declare the jexl module here